vfs_test.go

```diff
@@ -174,3 +164,108 @@ func TestVFSFH(t *testing.T) {
 	require.NoError(t, r.Close())
 	assert.ElementsMatch(t, b, bRead)
 }
+
+// TestVFSLs validates vfs LsDir operation is successful
+func TestVFSLs(t *testing.T) {
+	config, err := NewConfig()
+	require.NoError(t, err)
+
+	context, err := NewContext(config)
+	require.NoError(t, err)
+
+	vfs, err := NewVFS(context, config)
+	require.NoError(t, err)
+
+	tmpPath := filepath.Join(t.TempDir(), "somedir")
+	tmpPath2 := filepath.Join(tmpPath, "somedir2")
+	tmpPath3 := filepath.Join(tmpPath, "somedir3")
```

Comment:
Is the third one of these necessary? It seems like purely a duplicate of the "somedir2" case (which itself might be better renamed "subdir")

Reply:
subdir ok